Nushell update (#43)
MercuryTechnologies/nix-your-shell: README.md

### Nushell

> [!IMPORTANT]

> > Nushell version >=0.87.0 is required

Reviewer: Maybe we should detect this in `nix-your-shell`?

source nix-your-shell.nu
```

To generate the `nix-your-shell.nu` file:

Either manually generate it:

```nu
nix-your-shell nu | save $env.XDG_CONFIG_HOME/nushell/nix-your-shell.nu
```

> Or better yet, ensure it's kept updated alongside `nix-your-shell`. Populate file contents through home-manager:

Reviewer: ```suggestion
Or ensure it's kept updated alongside `nix-your-shell` by populate the file with [`home-manager`][home-manager]:

[home-manager]: https://nix-community.github.io/home-manager/
```
